src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -286,8 +287,37 @@ private bool TryGetTextForContextualKeyword(SyntaxToken token, out string text)
             text = null;
             return false;
         }
+        private static bool TryGetTextForCombinationKeyword(SyntaxToken token, out string text)
+        {
+            // Contextual keywords can appear in any order, and the user could initiate help from either keyword
+            // so to keep the actual checks simple we just check all 4 combinations
+            var previousToken = token.GetPreviousToken();
+            var nextToken = token.GetNextToken();
+            return TryGetTextForCombinationKeyword(token, previousToken, out text) ||
+                TryGetTextForCombinationKeyword(token, nextToken, out text) ||
+                TryGetTextForCombinationKeyword(previousToken, token, out text) ||
+                TryGetTextForCombinationKeyword(nextToken, token, out text);
+        }
+
+        private static bool TryGetTextForCombinationKeyword(SyntaxToken token1, SyntaxToken token2, out string text)
+        {
+            if (token1.Kind() == SyntaxKind.PrivateKeyword && token2.Kind() == SyntaxKind.ProtectedKeyword)
+            {
+                text = "privateprotected_CSharpKeyword";
+                return true;
+            }
+
+            if (token1.Kind() == SyntaxKind.ProtectedKeyword && token2.Kind() == SyntaxKind.InternalKeyword)
+            {
+                text = "protectedinternal_CSharpKeyword";
+                return true;
+            }
+
+            text = null;
+            return false;
+        }
```

Comment:
Will that still work if the developer swapped the keywords? e.g. used `internal protected` instead of `protected internal`

Reply:
Yep, this PR has tests for the keywords appearing in any order, and F1 being invoked for the first or second keyword, for both VB and C#. The method that calls this one has every permutation.